controllers/openshift_controller.go

```diff
@@ -1120,6 +1120,17 @@ func (r *KataConfigOpenShiftReconciler) processKataConfigInstallRequest() (ctrl.
 			return reconcile.Result{Requeue: true, RequeueAfter: 15 * time.Second}, err
 		}
 
+		// create PeerPodConfig CRD and runtimeclass for peerpods
+		if r.kataConfig.Spec.EnablePeerPods {
+			err = r.enablePeerPodsMiscConfigs()
+			if err != nil {
+				r.Log.Info("Enabling peerpodconfig CR, runtimeclass etc", "err", err)
+				// Give sometime for the error to go away before reconciling again
+				return reconcile.Result{Requeue: true, RequeueAfter: 15 * time.Second}, err
```

Comment:
I would suggest that in a situation where we know nothing about the nature or cause of an error, a more appropriate handling would be to rely on the exponential back-off provided by `reconcile.Result{Requeue: true}` instead of a specific timeout which, considering the lack of information, has to be basically arbitrary.

But this will be a subject of a future PR to unify our reconciler return values so no need to change it now.